docs/user_manual.md

```diff
@@ -139,6 +140,33 @@ A current best practice is to save all your job creation parameters in
 version-controlled files in your project directory. This allows you to tie
 your helios job params to changes in your application code.
 
+### Health Checks
+
+When a job is started, Helios can optionally run a health check before registering your service with
+nameless. This prevents the service from receiving traffic before it is ready. After the container
+starts, Helios will execute the health check as follows.
+
+* Begin executing health checks, using exponential backoff from 1 second to 30 seconds.
```

Comment:
For some reason, I found "using exponential backoff from 1 second to 30 seconds" confusing. Maybe "using exponential backoff that ranges randomly from 1 to 30 seconds."?